src/packages/@workspaces/features/tab-bar/components/tab/Tab.svelte

```diff
@@ -126,9 +126,7 @@
       {/if}
       <span
         class="font-weight-normal ms-1 {!tab.isActive ? 'request-text' : ''}"
-        style={`font-size: 12px; font-family: Roboto; color: ${
-          tab.isActive ? "#fff;" : "#8A9299;"
-        }`}
+        style={`font-size: 12px; font-family: Roboto; color:  var(--text-secondary-100)`}
```

Comment:
for font size use class "text-fs-12",
font family roboto globally integrated, not required here